### PR TITLE
Merge Nomi's Branch to include User Profiling

### PR DESCRIPTION
**Pull request by mohsinht**
@Muhammad1Nouman let us know which changes are the most important ones and which are causing conflicts in the live code.

### CODE REVIEW

File: config/keys.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 module.exports = {
   mongoURI: process.env.MONGO_URI,
-  secretOrKey: 'secret'
+  secretOrKey: 'secret',
+  jwtSecret:"mysecret"
```
Comment (ahmdhsn-dev) [inline config/keys.js:4]: Another secret key masly kry gi aik hi use kro jo pehly sa ha

**Comment by Muhammad1Nouman** [comment]
Exactly aik hi honi chiya aik reomove kr do

**Review by ahmdhsn-dev — APPROVED** [review]

**Review by mohsinht — COMMENTED** [review]
There's a problem in deployment:

```
-----> Node.js app detected
       
-----> Creating runtime environment
       
       NPM_CONFIG_LOGLEVEL=error
       NODE_ENV=production
       NODE_MODULES_CACHE=true
       NODE_VERBOSE=false
       
-----> Installing binaries
       engines.node (package.json):  10.15.3
       engines.npm (package.json):   unspecified (use default)
       
       Resolving node version 10.15.3...
       Downloading and installing node 10.15.3...
       Using default npm version: 6.4.1
       
-----> Restoring cache
       - node_modules
       
-----> Installing dependencies
       Installing node modules (package.json + package-lock)
       npm ERR! Unexpected string in JSON at position 490306 while parsing '{
       npm ERR!   "name": "taskbarter.com",
       npm ERR!   "version'
       
       npm ERR! A complete log of this run can be found in:
       npm ERR!     /tmp/npmcache.WDZwV/_logs/2019-08-05T10_39_33_141Z-debug.log
-----> Build failed
       
       We're sorry this build is failing! You can troubleshoot common issues here:
       https://devcenter.heroku.com/articles/troubleshooting-node-deploys
       
       If you're stuck, please submit a ticket so we can help:
       https://help.heroku.com/
       
       Love,
       Heroku
       
 !     Push rejected, failed to compile Node.js app.
 !     Push failed
```

**Comment by Muhammad1Nouman** [comment]
I just have create file UserInfo.js in profile section added some CSS in client style.css.
In models added new file named profileDetails
In routes/api/users.js I have made some changes from line to 270 to 274 in Test-Brach

**Pull request merged**